tutorial/source/intro_part_ii.ipynb

```diff
@@ -246,32 +246,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 10,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "a =  0.6285385489463806\n",
-      "b =  9.107474327087402\n"
+      "a =  9.107474327087402\n",
```

Comment:
I'm not able to see this rendered in github, but just a heads up that the output here seems to have been inter-changed. Is that okay?

Reply:
good catch!  all i did was rerun the cells. this seems to be another mistake, if you look at the [original graph](http://pyro.ai/examples/intro_part_ii.html#Next-Steps) `a` should be ~9.1 and `b` should be ~0.6. it must have been switched sometime and the cell wasnt rerun.

Reply:
Good to know.